test/units/testsuite-79.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -ex
+set -o pipefail
+
+if ! test -d /proc/pressure ; then
+    echo "kernel too old, has no PSI." >&2
+    echo OK >/testok
+    exit 0
+fi
+
+systemd-analyze log-level debug
+
+CGROUP=/sys/fs/cgroup/"$(systemctl show testsuite-79.service -P ControlGroup)"
+test -d "$CGROUP"
+
+if ! test -f "$CGROUP"/memory.pressure ; then
+    echo "No memory accounting/PSI delegated via cgroup, can't test." >&2
+    echo OK >/testok
+    exit 0
+fi
+
+UNIT="test-mempress-$RANDOM.service"
+SCRIPT="/usr/local/bin/mempress-$RANDOM.sh"
```

Comment:
Use /tmp/ for stuff generated on the fly

Reply:
Not an option. This is for a DynamicUser=1 service, and that implies PrivateTmp=1, hence /tmp is gonna be empty.

Reply:
/run then? We generally avoid writing to the image, as they are shared

Reply:
they are shared sure, but this uses a randomized name and deletes the file after, so this should be OK?

Reply:
executables in /run are icky, that's all